src/main/java/de/blablubbabc/paintball/Paintball.java

```diff
@@ -1107,6 +1108,14 @@ public void onEnable() {
 				Log.info("Plugin 'Vault' not found. Additional vault-economy-reward features disabled.");
 			}
 		}
+		// PlaceholderAPI
+		Plugin papi = getServer().getPluginManager().getPlugin("PlaceholderAPI");
+		if (papi != null && papi.isEnabled()) {
+		    Log.info("Plugin 'PlaceholderAPI' found. Using it now.");
+		    new PaintballPlaceholders(this).register();
```

Comment:
Does this need to react to and handle reloads of the PlaceholderAPI plugin?
Also, does this need to unregister the placeholderAPI extension again during shutdown, including when the paintball plugin is reloaded?

Reply:
It is marked persistent in the class, so it should be fine with PAPI reloads. I didn't think about what happens when the paintball plugin is reloaded though, so I'll have to look at that